src/assets/i18n/en.json5

```diff
@@ -3828,6 +3828,8 @@
 
   "submission.sections.general.add-more": "Add more",
 
+  "submission.sections.general.cannot_deposit": "Deposit cannot be completed due to missing mandatory information.<br>Please add them for final submission.",
```

Comment:
Tiny change, I'd recommend rewording this to say:
`Deposit cannot be completed due to errors in the form.<br>Please fill out all required fields to complete the deposit.`

Reply:
@tdonohue  done